package.json

```diff
@@ -85,7 +85,7 @@
     "babel-preset-env": "^1.6.1",
     "docusaurus-init": "^1.0.2",
     "dotenv": "^6.0.0",
-    "eyes.selenium": "0.0.83",
+    "eyes.selenium": "0.0.84",
```

Comment:
Looks like this isn't a valid version: https://www.npmjs.com/package/eyes.selenium